eval.py

```diff
@@ -135,9 +175,13 @@ def read_images(image_dir: str) -> Dict[str, Image.Image]:
 def read_prompts_for_images(images_by_filename: Dict[str, Image.Image], prompts_path: str):
     images = []
     prompts = []
+
     with open(prompts_path, "r") as f:
         prompts_by_image_filename = json.load(f)
-    for image_filename, prompt in prompts_by_image_filename.items():
+        # It's important to sort the filenames for pairwise similarity metrics.
+        prompts_by_image_filename = sorted(prompts_by_image_filename.items(), key=lambda x: x[0])
+
+    for image_filename, prompt in prompts_by_image_filename:
```

Comment:
didn't realize you could unpack this without explicitly calling out `prompts_by_image_filename.items()`

Reply:
I do on line 182